File: module1/Bai tap html javascript keystudy/Baitapkeystudy/Employee.js
class Employee {
    setName(name){
        this.name=name;
    }
    getName(){
        return this.name;
    }
    setBirthDay(birthday){
        this.birthday=birthday;
    }
    getBirthDay(){
        return this.birthday;
    }

    getPhone(){
        return this.phone;
    }
    setPhone(phone){
        this.phone=phone;
    }
    setEmail(email){
        this.email=email;
    }
    getEmail(){
        return this.email;
    }
    setCmnd(cmnd){
        this.cmnd=cmnd;
    }
    getCmnd(){
        return this.cmnd;
    }
    getLevel(){
        return this.level;
    }
    setLevel(level){
        this.level=level;
    }

    setLocation(location){
        this.location=location;
    }
    getLocation(){
        return this.location;
    }
    getSalary(){
        if(this.getLocation()==="Manager")
        {
            this.salary=500;
        }else if(this.getLocation()==="Sale"){
            this.salary=300
        }else if(this.getLocation()==="Markerting"){
            this.salary=200;
        }
        return this.salary;
    }
}

File: module 2/src/introductionToJava/baitapstackvaqueue/tochucdulieuhoply/Employee.java
package introductionToJava.baitapstackvaqueue.tochucdulieuhoply;

public class Employee {
    private String name;
    private String birthDay;
    private String gender;

    public Employee(String name, String birthDay, String gender) {
        this.name = name;
        this.birthDay = birthDay;
        this.gender = gender;
    }

    public Employee() {
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getBirthDay() {
        return birthDay;
    }

    public void setBirthDay(String birthDay) {
        this.birthDay = birthDay;
    }

    public String getGender() {
        return gender;
    }

    public void setGender(String gender) {
        this.gender = gender;
    }

    @Override
    public String toString() {
        return "Employee name is" +this.getName()+" Gender"+this.getGender()+" Birthday"+this.getBirthDay();
    }
}
